src/Selenium2Library/__init__.py

```diff
@@ -245,3 +262,7 @@ def _current_browser(self):
                       'use "Selenium2Library.browser" instead.',
                       DeprecationWarning)
         return self.browser
+
+    def _get_domains_visited(self):
+        assert(self._domain_tracking, "Must enable domain tracking to get the domains visited.")
```

Comment:
Using plain assert raises assertion error, which is not good idea. Either ValueError or create custom error class. Most likely later one would be better.